### PR TITLE
Make vibe.web.web.redirect similar to HTTPServerResponse.redirect

### PR DESCRIPTION
**Pull request by wilzbach**
Provide the same API as  `HTTPServerResponse.redirect`

### CODE REVIEW

File: web/vibe/web/web.d [hunk 1]
```diff
@@ -378,6 +378,12 @@ void redirect(string url)
 	ctx.res.redirect(fullurl);
```
Comment (thaven) [inline web/vibe/web/web.d]: I suppose `status` should be passed here....

File: web/vibe/web/web.d [hunk 2]
```diff
@@ -378,6 +378,12 @@ void redirect(string url)
...
 }
 
+/// ditto
+void redirect(URL url, int status = HTTPStatus.Found)
+@safe {
+	redirect(url.toString);
```
Comment (thaven) [inline web/vibe/web/web.d]: ... and here.

File: web/vibe/web/web.d (hunk 1 above)
Comment (s-ludwig) [reply web/vibe/web/web.d]: And `found` instead of `Found` ("deprecated")

**Comment by wilzbach** [comment]
Finally found time to rebase this PR - sorry for the forgetting to pass on `status`. Silly :S

**Pull request merged**